Xamarin.Forms.Core/ListView.cs

```diff
@@ -210,7 +212,8 @@ public SeparatorVisibility SeparatorVisibility
 			set { SetValue(SeparatorVisibilityProperty, value); }
 		}
 
-		internal ListViewCachingStrategy CachingStrategy { get; private set; }
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Yep, got that. I'm suggesting we change how we implement the interfaces (not whether we need them). I'd suggest moving from implicit implementation to explicit implementation with the addition of the EditorBrowser attribute. That way our callsites don't go through the interface and so are cleaner. Customers could do the same but will prob go through the interfaces because that is how they'll get isense as well as know that we're supporting the method.